api/conf/conf.json

```diff
@@ -10,6 +10,12 @@
     "dag-lib-path": "",
     "etcd": {
       "endpoints": "127.0.0.1:2379"
+    },
+    "log": {
+      "error_log": {
+        "level": "warn",
+        "file_path": "/dev/stdout"
+      }
```

Comment:
we'll remove this file finally, all right?

Reply:
sure.

Reply:
I think we can rebase the `v2.0` branch, and then we can remove this file